src/views/AccountingOverview/AccountingOverview.tsx

```diff
@@ -77,18 +77,22 @@ export const AccountingOverview = ({
             onboardingStepOverride={onboardingStepOverride}
           />
         )}
-        <div className='Layer__accounting-overview__summaries-row'>
-          <ProfitAndLoss.Summaries
-            stringOverrides={stringOverrides?.profitAndLoss?.summaries}
-            chartColorsList={chartColorsList}
-          />
-          {showTransactionsToReview && (
-            <TransactionToReviewCard
-              usePnlDateRange={true}
-              onClick={onTransactionsToReviewClick}
-            />
-          )}
-        </div>
+        <ProfitAndLoss.Summaries
+          stringOverrides={stringOverrides?.profitAndLoss?.summaries}
+          chartColorsList={chartColorsList}
+          slots={{
+            unstable_AdditionalListItems: showTransactionsToReview
+              ? [
+                  <TransactionToReviewCard
+                    key='transactions-to-review'
+                    usePnlDateRange={true}
+                    onClick={onTransactionsToReviewClick}
+                  />,
+                ]
+              : undefined,
+          }}
+          variants={{ size: 'lg' }}
```

Comment:
As I understood it, `sm` corresponds to what our components _currently_ default to. Setting this to `lg` would actually make the _default_ much larger, right? 

If I have that right, I'd much rather default to `sm` to preserve the current styling.

Reply:
One more important one: unless we expose the `variants` argument on `AccountingOverview`, customers won't actually be able to set this variant as in practice, they're using the `<AccountingOverview />` component to get this whole layout rather than using the `<ProfitAndLoss.Summaries />` where they could actually set the styleing.